docs/en_US/test-report.md

```diff
@@ -414,7 +416,9 @@ recv(51064): total=24866633, rate=1204424(msg/sec)
 
 recv(52000): total=25000000, rate=133367(msg/sec)
 
-QoS 1:
+</details>
+
+<details><summary>QoS 1</summary>
```

Comment:
How do u think about change it to `<summary>Receiving the QoS 1 message side</summary>`

Reply:
we can rephrase it later